Patch adapted from [1]:

	musl doesn't expose these sysconf values, so just use the loop
	below that reads them out of sysfs coherency_line_size only

[1] https://github.com/chimera-linux/cports/blob/114b576aee3ebc25b48ac120da1a1093dcc4365f/user/pcsx2/patches/cache.patch
--- a/common/Linux/LnxHostSys.cpp
+++ b/common/Linux/LnxHostSys.cpp
@@ -143,7 +143,11 @@
 size_t HostSys::GetRuntimeCacheLineSize()
 {
+	#if defined(__GLIBC__)
 	int l1i = sysconf(_SC_LEVEL1_DCACHE_LINESIZE);
 	int l1d = sysconf(_SC_LEVEL1_ICACHE_LINESIZE);
 	int res = (l1i > l1d) ? l1i : l1d;
+	#else
+	int res = 0;
+	#endif
 	for (int index = 0; index < 16; index++)
 	{
